packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -81,6 +86,28 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds the expansion panels' indicator.
+  ///
+  /// If null, [ExpandIcon] will be used as the expansion icon by default.
+  ///
+  /// {@tool sample}
+  ///
+  /// ```dart
+  /// ExpansionPanel(
+  ///  isExpanded: true,
+  ///  headerBuilder: (context, isExpanded) =>
+  ///      Text("This is ExpansionPanel's header"),
```

Comment:
Prefer single quote over double quote
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-single-quotes-for-strings